react/Checkbox/Checkbox.test.js

```diff
@@ -1,126 +1,61 @@
-import chai, { expect } from 'chai';
-import sinon from 'sinon';
-import sinonChai from 'sinon-chai';
 import React from 'react';
-import {
-  Simulate,
-  renderIntoDocument,
-  scryRenderedDOMComponentsWithClass,
-  findRenderedDOMComponentWithClass
-} from 'react-dom/test-utils';
-import { createRenderer } from 'react-test-renderer/shallow';
-import SyntheticEvent from 'react-dom/lib/SyntheticEvent';
-import { findAllWithClass } from 'react-shallow-testutils';
-import Checkbox from './Checkbox';
-
-chai.use(sinonChai);
-
-const eventMatcher = sinon.match.instanceOf(SyntheticEvent);
+import { shallow } from 'enzyme';
```

Comment:
So you're using render so `CheckMarkIcon` doesn't get rendered in the snapshots?  I think it'd be good for everyone to agree on an approach to this.  In profile we are explicitly mocking any components that we don't want to render.  Personally I prefer this approach but I'm keen to hear other opinions on this